app/uk/gov/hmrc/apiplatformoutboundsoap/models/OutboundSoapMessage.scala

```diff
@@ -134,13 +134,13 @@ case class RetryingOutboundSoapMessage(
     codMessage: Option[String] = None,
     coeMessage: Option[String] = None,
     sentDateTime: Option[Instant] = None,
-    privateHeaders: List[PrivateHeader] = List()
+    privateHeaders: List[PrivateHeader]
   ) extends OutboundSoapMessage {
   override val status: SendingStatus = SendingStatus.RETRYING
 
-  def toFailed = FailedOutboundSoapMessage(globalId, messageId, soapMessage, destinationUrl, createDateTime, ccnHttpStatus, notificationUrl, codMessage, coeMessage)
+  def toFailed = FailedOutboundSoapMessage(globalId, messageId, soapMessage, destinationUrl, createDateTime, ccnHttpStatus, notificationUrl, codMessage, coeMessage, None, List())
 
-  def toSent = SentOutboundSoapMessage(globalId, messageId, soapMessage, destinationUrl, createDateTime, ccnHttpStatus, notificationUrl, codMessage, coeMessage, Some(Instant.now))
+  def toSent = SentOutboundSoapMessage(globalId, messageId, soapMessage, destinationUrl, createDateTime, ccnHttpStatus, notificationUrl, codMessage, coeMessage, Some(Instant.now), List())
```

Comment:
This isn't quite what I meant. The creation of an empty list is now simply taking place somewhere else. 
Currently, in main, this class has `privateHeaders` declared as `Option[List]` and that clearly conveys the fact that an instance is complete without the `privateHeaders` member. 
Unless there's a reason I don't know of `privateHeaders` should continue to be a `Option[List[PrivateHeader]]`